provider/machine_pool_resource.go

```diff
@@ -326,7 +327,6 @@ func (r *MachinePoolResource) Update(ctx context.Context, request tfsdk.UpdateRe
 		)
 		return
 	}
-	object := get.Body()
```

Comment:
Wasn't being in use apparently. Please correct me if I'm mistaken

Reply:
you're right
Also please remove line 348 (that it's comment out)